src/main/kotlin/io/github/kryszak/gwatlin/api/gamemechanics/model/skill/SkillType.kt

```diff
@@ -19,5 +20,12 @@ enum class SkillType {
     @SerialName("Utility")
     UTILITY,
     @SerialName("Weapon")
-    WEAPON
+    WEAPON,
+    @SerialName("Toolbelt")
+    TOOLBELT,
+    @SerialName("Monster")
+    MONSTER,
+    @SerialName("Pet")
+    PET,
+    ;
```

Comment:
Unnecessary semicolon here, we might think of plugging in some kind of kotlin linter to avoid things like this in future

Reply:
Good catch, removed. I've had good experience with Detekt, if you are open for recommendations.